src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateServiceError.swift

```diff
@@ -106,45 +106,39 @@ enum HealthCertificateServiceError: Error {
 			switch self {
 			case .publicKeyRegistrationFailed(let registrationError):
 				switch registrationError {
-				case .badRequest:
+				case .receivedResourceError(.badRequest):
 					return String(format: AppStrings.HealthCertificate.Overview.TestCertificateRequest.Error.clientErrorCallHotline, "PKR_400")
```

Comment:
Would it be possible to move the strings to DCCPublicKeyRegistrationError (also for the other error types? It would cleanup this switch.

Reply:
I tried to move the `LocalizedError` conformances to the separate errors now where possible. For some it was not possible because the are defined in the HealthCertificateToolkit and sometimes reused in other places.